data-processing/pulsar-flink/1.15.1.1/pulsar-flink-new.md

```diff
@@ -0,0 +1,509 @@
+- Pulsar SQL connector: StreamNative will merge the Pulsar SQL connector back to the Flink community in the coming months.
+
+For detailed differences and recommendations, see the following sections.
+
+### Components
+
+| Repository              | Pulsar DataStream source | Pulsar DataStream sink | Pulsar SQL source | Pulsar SQL sink |
+|-------------------------|--------------------------|------------------------|-------------------|-----------------|
+| apache/flink 1.14       | Yes                      |                        |                   |                 |
+| apache/flink 1.15       | Yes                      | Yes                    |                   |                 |
+| streamnative/flink 1.15 | Yes                      | Yes                    | Yes               | Yes             |
+
+### Release process, version, and documentation
+
+| Repository        | Release                                                  | Versioning                                                       | DataStream connector documentation                                                                                              | SQL connector documentation |
+|-------------------|----------------------------------------------------------|------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| apache/flink      | Managed by Flink community, following Flink release process | Standard Semantic Versioning, like 1.15.0                        | Maintained on [Official Flink Website](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/) | Not available yet           |
+| streamnative/flink | Managed by StreamNative, following a monthly release process.    | Use an extra digit on top of the Flink version, such as 1.15.1.1 | Not available yet, but similar to the [Official Flink Website](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/)                                                                  | Maintained in current documentation   |
+
+### Issue and Feature tracking
+
+- For the Pulsar DataStream connector, use [Flink JIRA](https://issues.apache.org/jira/projects/FLINK/issues) to submit bug reports and feature requests.
+- For the Pulsar SQL connector, since it's not merged back to the official Flink repository yet, create an [issue](https://github.com/streamnative/flink/issues) in the `streamnative/flink` repository to submit bug reports and feature requests. Once the SQL Connector is merged back to the official Flink repository, use the Flink JIRA as well.
```

Comment:
```suggestion
- For the Pulsar SQL connector, since it's not merged back to the official Flink repository yet, create an [issue](https://github.com/streamnative/flink/issues) in the `streamnative/flink` repository to submit bug reports and feature requests. Once the Pulsar SQL connector is merged back to the official Flink repository, use the Flink JIRA as well.
```